packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -165,10 +165,10 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         __Ownable_init();
         transferOwnership(_owner);
 
-        overhead = _overhead;
-        scalar = _scalar;
-        batcherHash = _batcherHash;
-        gasLimit = _gasLimit;
+        _setGasConfig({ _overhead: _overhead, _scalar: _scalar });
+
+        _setBatcherHash(_batcherHash);
+        _setGasLimit(_gasLimit);
 
         _setAddress(_unsafeBlockSigner, UNSAFE_BLOCK_SIGNER_SLOT);
```

Comment:
double check if this is emitting an event

Reply:
Add tests for `expectEmit` ?

Reply:
Tests are added